### PR TITLE
CAT-ACCESSIBILITY Basic preferences fragment for accessibility

### PR DESCRIPTION
**Pull request by 84n4n4**

### CODE REVIEW

**Comment by 84n4n4** [comment]
please run espresso tests

**Comment by 84n4n4** [comment]
test this please

**Comment by 84n4n4** [comment]
please run espresso tests

**Comment by 84n4n4** [comment]
please run espresso tests

File: catroid/src/main/java/org/catrobat/catroid/ui/BaseActivity.java [hunk 1]
```diff
@@ -60,6 +55,47 @@ protected void onCreate(Bundle savedInstanceState) {
...
+		}
+		if (sharedPreferences.getBoolean("settings_accessibility_category_icons", false)) {
+			getTheme().applyStyle(R.style.CategoryIconVisible, true);
+		}
+		if (sharedPreferences.getBoolean("settings_accessibility_category_icons_big", false)) {
+			getTheme().applyStyle(R.style.CategoryIconSizeLarge, true);
+		}
+
+		@AccessibilitySettingsFragment.FontStyle
+		String fontStyle = sharedPreferences.getString("settings_accessibility_font_style", "regular");
```
Comment (robertpainsi) [inline catroid/src/main/java/org/catrobat/catroid/ui/BaseActivity.java]: Maybe `AccessibilitySettingsFragment.REGULAR` instead of `"regular"`?

Comment (84n4n4) [reply catroid/src/main/java/org/catrobat/catroid/ui/BaseActivity.java]: maybe... :-P

**Pull request merged**